src/web3/store/transactionsSlice.ts

```diff
@@ -82,6 +83,7 @@ export function createTransactionsSlice<T extends BaseTx>({
         from: tx.from,
         to: tx.to,
         nonce: tx.nonce,
+        timestamp: tx.timestamp || Math.floor(Date.now() / 1000),
```

Comment:
How can be possible that it will be no timestamp in the tx?

Reply:
for example, now when using the fork for governance there is no timestamp, I think only in this case, but maybe not

Reply:
Ok, but are we sure that tx is already mined here? If not, then not having a timestamp is correct and we should not put anything there. Or we can do something like "creationTs". Or Maybe we need to .wait()?

Reply:
ok, I'll take a closer look

Reply:
after a little research, I saw that there is no timestamp in the tx, we need to get timestamp from block number, but we don't have block number when tx just created.

I suggest doing this: 

1) Pass the current date to the timestamp when tx just created. Since we need timestamp while the tx is pending.
2) Update tx timestamp at the moment when we update the statuses.

What do you think about this @kyzia551 @satanworker ?